src/config.rs

```diff
@@ -174,6 +179,8 @@ impl Config {
                 "CACHE_CONTROL_STALE_WHILE_REVALIDATE",
             )?,
 
+            full_page_cache: env("DOCSRS_FULL_PAGE_CACHE", true)?,
```

Comment:
I feel like there's probably a better name for this variable, something like `DOCSRS_CACHE_INVALIDATEABLE_RESPONSES` maybe?